client/my-sites/plugins/plugins-announcement-modal/index.jsx

```diff
@@ -14,14 +13,13 @@ const PluginsAnnouncementModal = () => {
 	const jetpackNonAtomic = useSelector(
 		( state ) => isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId )
 	);
-	const sitePlan = useSelector( ( state ) => getSitePlan( state, siteId ) );
 
 	if ( jetpackNonAtomic ) {
 		// Buying plugins is not yet available to self hosted sites.
 		return null;
 	}
 
-	if ( ! siteId || ! isStarter( sitePlan ) ) {
+	if ( ! siteId ) {
```

Comment:
Since the code underneath uses `selectedSiteUrl` would it be preferable checking against it instead of `siteId`?

Reply:
I agree, it is more coherent with the code below. Changing it